packages/primitives/popover/src/popover-arrow.directive.ts

```diff
@@ -0,0 +1,86 @@
+import { computed, Directive, effect, ElementRef, inject, input, Renderer2 } from '@angular/core';
+import { RdxPopoverContentToken } from './popover-content.token';
+import { RdxPopoverSide } from './popover.types';
+
+@Directive({
+    selector: '[rdxPopoverArrow]',
+    standalone: true
+})
+export class RdxPopoverArrowDirective {
```

Comment:
can be taken out in individual component/directive later. same code used in tooltip. just for code reusing

Reply:
sure, I've actually copied and changed some stuff from the tooltip codebase as that is the most similar functionality to the popover